src/backend/tcop/backend_startup.c

```diff
@@ -889,3 +888,12 @@ StartupPacketTimeoutHandler(void)
 {
 	_exit(1);
 }
+
+/*
+ * Wrapper for ProcessSSLStartup to handle direct SSL handshake
+ */
+int
+WrapperProcessSSLStartup(Port *port)
+{
+    return ProcessSSLStartup(port);
+}
```

Comment:
Instead of creating wrapper function just to use it different file, please change declaration of ProcessSSLStartup from static to extern directly.